Assets/Scripts/Source/Audio/AudioOptions.cs

```diff
@@ -29,12 +30,12 @@ public bool HasEndHandler
         }
     }
 
-    public float FadeInTime { get; private set; }
-    public float FadeOutTime { get; private set; }
-    public float Volume { get; private set; }
-    public bool Looping { get; private set; }
+    public float FadeInTime;
```

Comment:
Prefer public getter pattern + `[SerializeField]` on private vars (to preserve editor visibility, which was a great addition)!